backend/project-service/api/src/main/kotlin/Projects.kt

```diff
@@ -896,4 +903,23 @@ object Projects : CallDescriptionContainer("project") {
                 }
             }
         }
+
+    val searchProjectPaths = call<SearchProjectPathsRequest, SearchProjectPathsResponse, CommonErrorMessage>("findProjectPaths") {
+        auth {
+            access = AccessRight.READ
+        }
+
+        http {
+            method = HttpMethod.Get
+
+            path {
+                using(baseContext)
+                +"search-project-paths"
```

Comment:
I think just using `search` is sufficient here when coupled with a change of query parameter names.

```suggestion
                +"search"
```